SourceryTests/Generating/StencilTemplateSpec.swift

```diff
@@ -90,7 +90,7 @@ class StencilTemplateSpec: QuickSpec {
                     expect(generate("{{ type.MyClass.variables.1.typeName | capitalise }}")).to(equal("Myclass"))
                 }
 
-                it("checks for string in name") {
+                fit("checks for string in name") {
```

Comment:
I think this is a typo 😄

Reply:
Yeah, @SourceryBot caught it 😄 
Removed in e56fc88